Improve docs metadata (#405)
o1-labs/docs2: docs/zkapps/how-zkapps-work.mdx

---
title: How zkApps Work
hide_title: true

> description: Provides an in-depth explanation of how zkApps work, including their structure, the principles of zero-knowledge based smart contracts, the prover and verifier functions, the deployment and interaction processes, and the state management on-chain and off-chain.

Reviewer: ```suggestion
description: In-depth explanation of how zkApps work. The structure of a zkApp, zero knowledge (zk) principles, zero knowledge-based smart contracts, prover function, verifier function, how to deploy and interact with a zkApp, and on-chain and off-chain state management.
```
